src/NHapi.Base/Parser/ParserConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace NHapi.Base.Parser
+{
+    public class ParserConfiguration
+    {
+        public ParserConfiguration()
+        {
+            NonGreedyMode = false;
+        }
+
+        /// <summary>
+        /// If set to true (default is false), pipe parser will be put in non-greedy mode. This setting applies only to
```

Comment:
Can we add any missing xml doc tags to this please, will be useful in the future when we want to generate docs.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags